src/logged_in/components/dashboard/Patient.js

```diff
@@ -8,7 +8,7 @@ class Patient {
       age: 0,
       gender: 'male'
     };
-    this.coronaPositive = false;
+    this.coronaPositive = undefined;
```

Comment:
should be true by default for now (shir said)
```suggestion
    this.coronaPositive = true;
```